layers/meta-resin-raspberrypi/recipes-core/images/resin-image.bbappend

```diff
@@ -8,15 +8,25 @@ SDIMG_KERNELIMAGE_raspberrypi3-64 ?= "kernel8.img"
 # Customize resinos-img
```

Comment:
I would split this changes in another commit - for resin-image.

Reply:
yea splitting would be wiser

Reply:
I would tweak a little bit the commit log: "/mnt/boot" is a path on root. Which happens to be a mountpoint to boot partition but it's confusing. I would say that that it is placed in the root of the boot partition. Whatever root uses to mount that, it's another story.